### ISSUE TITLE
Linkedin sharing title/text seems deprecated

### ISSUE THREAD

**Issue by AngelaDu** [nygardk/react-share#550]
I think the package uses:
https://learn.microsoft.com/en-us/linkedin/marketing/community-management/shares/ugc-post-api?view=li-lms-unversioned&context=linkedin%2Fcompliance%2Fcontext&tabs=http

which is now deprecated, so linked in title/body etc do not get populated anymore

**Comment by rom212** [comment]
Same problem here

**Comment by g-hamilton** [comment]
Yeah me too. It seems the summary prop is being ignored? I just get a blank share dialogue with the source value as the share url rather than the actual share url and summary. Means I can't share a specific article - only my home page :(
